docs/migrate-to-jina3/migrate.md

```diff
@@ -0,0 +1,72 @@
+# Migration to Jina 3
+
+Jina 3 introduces a number of exciting features, but to be able to enjoy them, you will also have to make some
```

Comment:
Do we want to mention (and link to) some of the new features briefly? That'll get people excited to use 3.0 and will want to read the rest of this guide

Reply:
Definitely, but I don't know that we have such a link yet